index.js

```diff
@@ -83,9 +94,24 @@ export const config = {
 			'number'
 		],
 		default: 80
+	},
+	syntax: {
```

Comment:
As commented in the original issue, it shouldn't be an option, but just handled automagically.